src/copyright/ui/ajax-copyright-hist.php

```diff
@@ -100,7 +100,15 @@ function Output()
     else if($action=="update" || $action=="delete" || $action=="undo")
     {
       $id = GetParm("id", PARM_STRING);
-      list($upload, $item, $hash, $type) = explode(",", $id);
+      $getEachID = explode(",", trim($id, ','));
```

Comment:
```suggestion
      list($upload, $item, $hash, $type) = explode(",", $id);
      if (empty($upload) || empty($item) || empty($hash) || empty($type))
      {
          return new JsonResponse(array(...)...);
      }
```
list($upload, $item, $hash, $type) = explode(",", $id);

Reply:
The suggested code will lead to the below PHP Notice: 

```
PHP Notice:  Undefined offset: 1 in /usr/share/fossology/copyright/ui/ajax-copyright-hist.php on line 103
PHP Notice:  Undefined offset: 2 in /usr/share/fossology/copyright/ui/ajax-copyright-hist.php on line 103,
PHP Notice:  Undefined offset: 3 in /usr/share/fossology/copyright/ui/ajax-copyright-hist.php on line 103,
```

Reply:
what was your input? I.e. what was the value of `$id`?